packages/pn-personafisica-webapp/package.json

```diff
@@ -4,15 +4,14 @@
   "version": "2.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.10.4",
-    "@emotion/styled": "^11.10.4",
-    "@mui/icons-material": "5.11.11",
-    "@mui/lab": "5.0.0-alpha.100",
```

Comment:
be sure that removing this package, doesn't break the app.

this package is requested by mui-italia

Reply:
I'm not sure that it works, the module is installed because is a dependency of the other app.

If you try to install the package stand-alone, it will not work